src/main/java/mate/academy/bookstore/security/JwtAuthenticationFilter.java

```diff
@@ -18,6 +20,7 @@
 @Component
 @RequiredArgsConstructor
 public class JwtAuthenticationFilter extends OncePerRequestFilter {
+    public static final String AUTHORIZATION_SCHEMA_BEARER = "Bearer ";
```

Comment:
you use it only in this class
```suggestion
    private static final String AUTHORIZATION_SCHEMA_BEARER = "Bearer ";
```